recipe/multitenancy/utils.go

```diff
@@ -57,7 +57,15 @@ func makeTypeNormalisedInput(appInfo supertokens.NormalisedAppinfo, config *mult
 		config.ErrorHandlers.OnRecipeDisabledForTenantError = &onRecipeDisabledForTenantError
 	}
 
+	if config.GetDomainsForTenantId == nil {
+		config.GetDomainsForTenantId = func(tenantId string, userContext supertokens.UserContext) ([]string, error) {
+			return []string{}, nil
+		}
```

Comment:
this should not be done. Keep it nil if the user has not initialised it. Cause this way, the allowedList in the multi tenancy claim would not exist (saving space).

Reply:
Have removed the default implementation. The claim will return an error if this function is not provided.